EWBF/0.3.4b/.nvoc-miner.json

```diff
@@ -1,14 +1,14 @@
 {
 	"friendlyname": "ewbf",
 	"version": "0.3.4b",
-	"hcd": "miner --eexit 3 --fee $EWBF_PERCENT --pec --server ${!xpool} --user ${!xaddr}${!xwallet}${!xwork} --pass $MINER_PWD --port ${!xport} $OPTS",
+	"hcd": "miner --eexit 3 --fee $EWBF_PERCENT --pec --server ${!xpool} --user ${!xaddr}${!xwallet}${!xwork} --pass $MINER_PWD --port ${!xport}",
```

Comment:
Use ${!xfee} in place of $EWBF_PERCENT and $Z_EWBF_PERCENT, just added in 0miner. Every time you think to add here some variable which contains the miner name as derived from its containing folder name its somewhat ideally wrong and should be addressed differently for portability reasons ;-)